src/Microsoft.CodeQuality.Analyzers/Core/Microsoft.CodeQuality.Analyzers.csproj

```diff
@@ -14,6 +14,11 @@
     <InternalsVisibleTo Include="Microsoft.CodeQuality.Analyzers.UnitTests" />
     <PackageReference Include="Humanizer.Core" Version="$(HumanizerVersion)" />
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Update="Performance\MicrosoftPerformanceAnalyzersResources.resx">
```

Comment:
Please revert this change to project file. I believe the SDK already takes care of this...